exporter/opentelemetry-exporter-prometheus/pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 dependencies = [
   "opentelemetry-api ~= 1.12",
   # DONOTMERGE: confirm that this will becomes ~= 1.21 in the next release
-  "opentelemetry-sdk ~= 1.21.0.dev",
```

Comment:
Should become ~= 1.21, see the comment above. Also, can remove the the donotmerge comment

Reply:
Thanks for pointing that out, fixed.